src/components/newTodo/NewTodo.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import './newTodo.css';
+
+class NewTodo extends React.Component {
+  state = {
+    userId: 0,
+    title: '',
+    inputErrorMessage: false,
+    selectErrorMessage: false,
+  };
+
+  handleNewTaskName = (event) => {
+    this.setState({
+      title: event.target.value,
+      inputErrorMessage: false,
+    });
+  }
+
+  handleSelect = (event) => {
+    this.setState({
+      userId: Number(event.target.value),
+      selectErrorMessage: false,
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { title, userId } = this.state;
+    let error = false;
+
+    this.setState({
```

Comment:
one setState for one method,
write all code inside setState, use callback variant of it

Reply:
wasn't fixed